hbase-server/src/test/java/org/apache/hadoop/hbase/coprocessor/TestCoprocessorHost.java

```diff
@@ -41,6 +44,8 @@
   public static final HBaseClassTestRule CLASS_RULE =
       HBaseClassTestRule.forClass(TestCoprocessorHost.class);
 
+  private static final HBaseCommonTestingUtility TEST_UTIL = new HBaseCommonTestingUtility();
```

Comment:
this two lines related to `HBaseCommonTestingUtility` are the only changes from #4069